lib/manager/gitlabci/extract.ts

```diff
@@ -7,12 +7,17 @@ import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
 import type { GitlabPipeline } from './types';
 import { replaceReferenceTags } from './utils';
 
+const commentsRe = /^\s*#/;
+const whitespaceRe = /^(?<whitespace>\s*)/;
+const imageRe = /^(?<whitespace>\s*)image:\s*'?"?(?<image>[^\s'"]+|)'?"?\s*$/;
```

Comment:
```suggestion
const imageRe = /^(?<whitespace>\s*)image:\s+'?"?(?<image>[^\s'"]+|)'?"?\s*$/;
```

You've lost one. 😏 Next time you can simply apply my sugesstion 😉

Reply:
I have applied your suggestion :P 
but this regex has to match this case too: 
```yaml
image:
  name: hub.docker.io/myOrg/myImage
```

Reply:
It won't match that anyways, as we go line-by-line.

Reply:
ok, what about the updated regex above 🤔

Reply:
That didn't worked either, but the problem has been in another section.    
I wanted initially to change the regexes as little as possible, but as we have started to optimize them I have changed the others as well. 

Regarding the image regex I have gone with this: 
```
const imageRe = /^(?<whitespace>\s*)image:(?:\s+['"]?(?<image>[^\s'"]+)['"]?)?\s*$/;
```